fury-on-ovhcloud/infrastructure/terraform/README.md

```diff
@@ -0,0 +1 @@
+# COMING SOON
```

Comment:
You can use [`terraform-docs`](https://terraform-docs.io/) to generate a basic readme describing the terraform module 😉 

You can see a sample configuration file that we use here:
https://github.com/sighupio/fury-eks-installer/blob/master/modules/eks/.terraform-docs.yml

Reply:
Thx for the tip!
I used it for the main terraform script and the modules taht are used